app/sanitizers/user/parameter_sanitizer.rb

```diff
@@ -5,7 +5,7 @@ class ParameterSanitizer < Devise::ParameterSanitizer
       email
       password
       password_confirmation
-    )
+    ).freeze
```

Comment:
It is a recommendation from new cop `Style/MutableConstant` because it is actually possible to modify non-frozen constants in ruby. E.g.:

```ruby
class Foo
  DEFAULTS = [:a, :b]
end

default = Foo::DEFAULTS
default << :c
Foo::DEFAULTS #=> [:a, :b, :c]
```